TurkishFilter/sections/specific.txt

```diff
@@ -1,4 +1,11 @@
 !
+!
+! This section contains the list of domain-specific rules that block ads.
+!
+! Good: example.org###rek
```

Comment:
Basic rules are also allowed in all `specific.txt` files, should be added to examples.

Reply:
Added.